CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+## [2.20.0] - 2020-01-10
+
+## Added
+- Added/fixed podAntiAffinity rules for various roles
+- Set topologyKey for podAntiAffinity rule to kubernetes.io/hostname and made it configurable
```

Comment:
I wouldn't mention that it is configurable; we don't document how to configure it, and don't expect people to modify it, except (maybe) under guidance from our service/support people.